README.md

```diff
@@ -1,51 +1,48 @@
 
-## Applications are now open for Hackathon Travel Awards
+##  Is there any funding available to Brainhack attendees?
 
-The Open Science Special Interest Group is pleased to offer 20 travel awards that will include free registration at the hackathon plus $500 that will go towards covering part of the expenses incurred to attend the hackathon. For more information see this [page](./travel_awards.md).
+Yes. The OS-SIG is providing 50 mini-grants worth 100 USD each to help Brainhack attendees set up for online conferencing. Apply via the registration link.
 
-## Registration for the hackathon will open mid-March.
+In addition, the OS-SIG is committed to make the OHBM Brainhack more accessible. In particular, if the Brainhack fees make it impossible for you to attend the OHBM Brainhack, please send us an email at: ossig2019@inria.fr sharing some details about your situation. We will consider waving those fees as necessary.
 
-Soon after abstract acceptances are sent out, registration will open. In the past two years the hackathon was fully booked and we therefore recommend interested attendees to register early. Seats will be allocated on a first come first serve basis. In an effort to increase diversity at the event, we will reserve part of the seats to attendees who attend a hackathon for the first time or who come from traditionally under-represented groups in science or in past hackathons.
+## Do I need to register for the OHBM conference to attend the OHBM Brainhack?
+No.
+The main conference and the OHBM Brainhack are separate events and you can choose to register for Brainhack only.
 
-## Please contribute!
-
-Are there specific topics that you would like to see covered in the Hackathon training sessions? Please let us know using this [form](https://forms.gle/LbngA8xVDYj3Ep9W7).
+You do not need to be an OHBM member to register and the price of registration will be the same regardless of your membership status.
 
 ## For more information
-For most recent updates please see the [OS SIG website](https://ossig.netlify.com/), follow the hbm-hackathon channel on the [brainhack mattermost](https://mattermost.brainhack.org/signup_user_complete/?id=orpd9qqjb7gqpnwg5k1fdagrq) and follow us on twitter [@OHBMopen](https://twitter.com/ohbmopen).
+For most recent updates please go to [OHBM Brainhack 2020 webpage](https://ohbm.github.io/hackathon2020), read our [FAQs](https://ohbm.github.io/hackathon2020/FAQ/), go to the [OS SIG website](https://ossig.netlify.com/), follow the hbm-hackathon channel on the [brainhack mattermost](https://mattermost.brainhack.org/signup_user_complete/?id=orpd9qqjb7gqpnwg5k1fdagrq) and follow us on twitter [@OHBMopen](https://twitter.com/ohbmopen).
 
-## Website template
-Websate is based on the [Zeppelin](https://github.com/gdg-x/zeppelin)  template
-developed by GDG-X and the version adapted by [Brainhack Global Finland 2018](https://brainhackfinland.github.io/2018/).
 
 ## Sponsors
```

Comment:
Actually I will remove this part as we now have a proper sponsors page on the website. :-)